src/web_service/cookie.h

```diff
@@ -0,0 +1,98 @@
+// SPDX-FileCopyrightText: 2023 Yuzu Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include <ctime>
+#include <regex>
+#include <string>
+#include <utility>
+#include <vector>
+
+namespace WebService {
+
+class Cookie {
+public:
+    explicit Cookie(std::string& name, std::string& value);
+    Cookie() = default;
+    ~Cookie() = default;
+
+    enum RawForm { NameAndValueOnly, Full };
```

Comment:
```suggestion
    enum class RawForm { NameAndValueOnly, Full };
```

Realized I forgot to make a comment for this one